Formula/java-service-wrapper.rb

```diff
@@ -23,9 +24,17 @@ def install
     # Default javac target version is 1.4, use 1.6 which is the minimum available on openjdk@11
     system "ant", "-Dbits=64", "-Djavac.target.version=1.6"
     libexec.install "lib", "bin", "src/bin" => "scripts"
+    if OS.mac?
+      if Hardware::CPU.arm?
+        ln_s "libwrapper.dylib", libexec/"lib/libwrapper.jnilib"
+      else
+        ln_s "libwrapper.jnilib", libexec/"lib/libwrapper.dylib"
+      end
```

Comment:
Won't these symlinks be installed into `buildpath`?

Reply:
That's the idea, right?
```
$ ls -l /opt/homebrew/Cellar/java-service-wrapper/3.5.48_1/libexec/lib/
total 448
-rwxr-xr-x 1 aho admin  95565 Jan 10 18:33 libwrapper.dylib*
lrwxr-xr-x 1 aho admin     16 Jan 10 18:33 libwrapper.jnilib -> libwrapper.dylib*
-rw-r--r-- 1 aho admin 132746 Jan 10 18:33 wrapper.jar
-rw-r--r-- 1 aho admin  68944 Jan 10 18:33 wrapperdemo.jar
-rw-r--r-- 1 aho admin 150085 Jan 10 18:33 wrappertest.jar
-rw-r--r-- 1 aho admin   1542 Jan 10 18:33 wrappertest2.jar
```
Or am I missing something?

Reply:
It is? But `buildpath` is ephemeral, so how will other formulae make use of the symlink?